### ISSUE TITLE
F5 appsvcs throwing 404 when the admin user is disabled (URI:/mgmt/shared/appsvcs/info)

### ISSUE THREAD

**Issue by uts09** [F5Networks/f5-appsvcs-extension#650]
### Environment
 * Application Services Version: 3.36.0/3.37.0/3.38.0/3.39.0
 * BIG-IP Version:  15.1.6.1 (Hosted on Azure)

### Summary
F5 AppSvcs is throwing 404 when the admin user is disabled. URI is /mgmt/shared/appsvcs/info which is required by the F5 CIS to function.

### Steps To Reproduce

Disable the admin user by navigating to System -> Platform ->User Administration -> Disable default admin, use alternate.
Optionally restart the services by running bigstart restart restjavad restnoded

Issue is not seen when the admin user is enabled.

### Observe the following error response:
```
curl -k -s -u 'testuser:pass' https://10.222.22.4/mgmt/shared/appsvcs/info  | jq .
{
  "code": 404,
  "message": "",
  "referer": "10.255.11.4",
  "errorStack": []
}

```

### Expected Behavior
```
curl -k -s -u 'testuser:pass' https://10.222.22.4/mgmt/shared/appsvcs/info | jq .
{
  "version": "3.38.0",
  "release": "4",
  "schemaCurrent": "3.38.0",
  "schemaMinimum": "3.0.0"
}
```

### Actual Behavior
GET request to /mgmt/shared/appsvcs/info is failing.

```
curl -k -s -u 'testuser:pass' https://10.222.22.4/mgmt/shared/appsvcs/info  | jq .
{
  "code": 404,
  "message": "",
  "referer": "10.255.11.4",
  "errorStack": []
}


```

```
tail -f restnoded/restnoded.log

Tue, 20 Sep 2022 12:02:03 GMT - finest: socket 193 opened
Tue, 20 Sep 2022 12:02:03 GMT - severe: [RestOperationDispatcher] 'shared/appsvcs/info' not found.
Tue, 20 Sep 2022 12:02:03 GMT - severe: [ErrorHandlingModule] RestOperation failed: "/shared/appsvcs/info". {"code":404,"message":"","referer":"10.255.11.4","originalRequestBody":"","errorStack":[]}
Tue, 20 Sep 2022 12:02:04 GMT - config: [RestWorker] /shared/appsvcs has started. Name:RestWorker
Tue, 20 Sep 2022 12:02:11 GMT - finest: socket 193 closed
Tue, 20 Sep 2022 12:02:14 GMT - finest: socket 194 opened

```

**Comment by sunitharonan** [comment]
Hi @uts09, does testuser has administrative role based access? Please find this article for prerequisites & requirements for using BIG-IP AS3. https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/userguide/prereqs.html

**Comment by uts09** [comment]
Yes, testuser has admin privileges.

**Comment by sunitharonan** [comment]
Thank you. I have added this issue to our internal product backlog as AUTOTOOL-3470

**Comment by sunitharonan** [comment]
HI @uts09, would like to put a name to AUTOTOOL-3470 internally. Please can you reach out to us at automation_toolchain_pm@f5.com so we can prioritize.

**Comment by seattlevine** [comment]
This will be in 3.42, but you will need to `bigstart restart restjavad restnoded` after disabling the default admin user

**Comment by sunitharonan** [comment]
This has been resolved in the 3.42.0 release of AS3.
